docs/optimization_guide/nncf/ptq/quantization_w_accuracy_control.md

```diff
@@ -51,14 +51,24 @@ Now, you can run quantization with accuracy control. The following code snippet
       :fragment: [quantization]
 
 
-``max_drop`` defines the accuracy drop threshold. The quantization process stops when the degradation of accuracy metric on the validation dataset is less than the ``max_drop``.
+* ``max_drop`` defines the accuracy drop threshold. The quantization process stops when the degradation of accuracy metric on the validation dataset is less than the ``max_drop``. The default value is 0.01.
+
+* ``drop_type`` defines how the accuracy drop will be calculated: ``ABSOLUTE`` (used by default) or ``RELATIVE``.
+
+``nncf.quantize_with_accuracy_control()`` API supports all the parameters from :doc:`Basic 8-bit quantization <basic_quantization_flow>` API and you can also use them to quantize a model with accuracy control and a custom configuration.
+
```

Comment:
I think it is worth adding what to do with model later. You already have statement in basic flow that model could be serialized (I have comment for that section also - example missing there).

Reply:
I've added code example and some text. Please check it.